deploy/kustomizations/bmo-ironic-keepalived/keepalived_patch.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metal3-keepalived
```

Comment:
That name will not match anything. You actually only need one keepalived_patch.yaml, you could have in one place and refer to it from multiple kustomizations instead of duplicating